.circleci/config.yml

```diff
@@ -72,7 +72,35 @@ jobs:
       - slack/notify:
           event: fail
           channel: << pipeline.parameters.alerts-slack-channel >>
-          template: basic_fail_1
+          custom: |
```

Comment:
given we're using the hmpps circle ci orb, is there no way to use their template definition directly (slack_failed.tpl.json)?

Looking at the following example i wonder what happens if we remove the template definition altogether - will it fall back to version in hmpps circle ci orb?

https://github.com/ministryofjustice/hmpps-circleci-orb/blob/main/src/examples/slack_notification.yml

Reply:
Every call to the Slack Notify task in the Orb [defines the template explicitly](https://github.com/ministryofjustice/hmpps-circleci-orb/blob/main/src/jobs/npm_outdated.yml#L32) using the `<< include(template.json) >>` syntax, but this syntax is only available to Orb definitions ([docs](https://circleci.com/docs/orb-concepts/#file-include-syntax).

I don’t think the template name is exposed beyond the orb either. The Orb’s deploy success task actually defines its own template in place [here](https://github.com/ministryofjustice/hmpps-circleci-orb/blob/main/src/jobs/deploy_env.yml#L130), which makes me think it has to be defined explicitly.

Reply:
Cool, thanks for looking into it